plugin/src/cli_utils/supportability/mod.rs

```diff
@@ -0,0 +1,131 @@
+use std::path::PathBuf;
+use supportability::operations::SystemDumpArgs;
+
+pub mod cli;
+pub mod dump;
+
+/// Collects state & log information of mayastor services running in the system and dump them.
+#[derive(Debug, Clone, clap::Args)]
+pub struct SupportArgs {
+    /// Specifies the timeout value to interact with other modules of system
+    #[clap(global = true, long, short, default_value = "10s")]
+    timeout: humantime::Duration,
+
+    /// Period states to collect all logs from last specified duration
+    #[clap(global = true, long, short, default_value = "24h")]
+    since: humantime::Duration,
+
+    /// Endpoint of LOKI service, if left empty then it will try to parse endpoint
+    /// from Loki service(K8s service resource), if the tool is unable to parse
+    /// from service then logs will be collected using Kube-apiserver
+    #[clap(global = true, short, long)]
+    loki_endpoint: Option<String>,
+
+    /// Endpoint of ETCD service, if left empty then will be parsed from the internal service name
+    #[clap(global = true, short, long)]
+    etcd_endpoint: Option<String>,
+
+    /// Output directory path to store archive file
+    #[clap(global = true, long, short = 'd', default_value = "./")]
+    output_directory_path: String,
+
+    /// Kubernetes namespace of openebs service
+    #[clap(global = true, long, short = 'n', default_value = "openebs")]
+    namespace: String,
```

Comment:
We're not assuming defaults for the namespace typically. The namespace is passed down from the caller functions, it doesn't need to be a clap arg.

Reply:
This has been the behaviour of the system dump all these while for mayastor so kept the same behaviour here. This is just to add a layer on convenience as mostly users would install it in openebs ns, so they don't need to pass -n openebs all the time.

Reply:
The plugin behaviour is to use the default namespace. The openebs plugin doesn't have this behaviour of assuming the namespace of the deployment. This is only exclusive to the mayastor plugin.
cc: @tiagolobocastro @abhilashshetty04

Reply:
Why did we decide to change the behavior for openebs then? Should it be same for all plugins, even the previous openebs ctl used to work this way.

Reply:
It's very unlikely that someone will install it in default namespace.

Reply:
No, let's not do this, we had already decided on this for mayastor v3 and openebs (since it's a new plugin, so no breaking changes), namespace is a required parameter

Reply:
Anyway I have kept the behaviour same as the current plugin. Have reverted this. PTAL

Reply:
I mean, so should have no default and user must specify

Reply:
Done.